src/components/game/layout.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import * as ReactRouter from 'react-router';
+import { bindActionCreators } from 'redux';
+import {move, start} from '../../actions/game';
+import {DOWN, LEFT, RIGHT, UP} from '../../constants';
+
+interface IProps {
+  game: any;
+  dispatch: Function;
+}
+
+class GameLayout extends React.Component<IProps, {}> {
+  public componentDidMount() {
+    this.props.dispatch(start());
+    document.addEventListener('keyup', this.determineMove);
```

Comment:
You should attach event and remove it from document once it's unmounted. Here is reference of component lifecycle: https://www.tutorialspoint.com/reactjs/reactjs_component_life_cycle.htm